### PR TITLE
Add missed features for Samsung Internet

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
Last time I did an update I accidentally missed the CSS folder, this adds those changes and brings in a few changes that had been made in the meantime.

This is generated using the script.

### CODE REVIEW

File: css/properties/aspect-ratio.json [hunk 1]
```diff
@@ -59,7 +59,9 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "12.0",
+              "partial_implementation": true,
+              "notes": "Samsung Internet 79 adds internal support only for mapped values"
```
Comment (foolip) [inline css/properties/aspect-ratio.json:64]: The version number here is wrong. I guess the script copies notes and replaced the browser name, but not version numbers?

File: css/properties/text-rendering.json [hunk 2]
```diff
@@ -52,8 +52,11 @@
               "version_added": "4.2"
             },
             "samsunginternet_android": {
-              "version_added": false,
-              "notes": "This property is only supported on Windows and Linux. Samsung Internet is not on Windows or Linux."
+              "version_added": "1.0",
+              "notes": [
+                "This property is only supported on Windows and Linux.",
```
Comment (foolip) [inline css/properties/text-rendering.json:57]: The old notes suggest that this shouldn't be listed as supported. That chrome_android is listed as supported is pretty weird too. Can you test whether it's actually supported, and update notes if so?

**Review by foolip — COMMENTED** [review]
I skimmed the changes and they mostly seem plausible, but two notes seem wrong.

File: css/properties/text-rendering.json (hunk 2 above)
Comment (AdaRoseCannon) [reply css/properties/text-rendering.json:57]: Whoops that was a git mistake on my part that should've been reverted, thanks for the spot!

**Review by foolip — APPROVED** [review]
Looks good, thanks @AdaRoseCannon!

**Comment by lolopedigo** [comment]
Hi there. My name is lauren.  I am being bombarded with these emails
and I have 2 phones that someone has somehow got into.  They have
destroyed my sanity and my qiulity of life and now they have taken
thousands of dollars from me.  I am getting encrypted messages from
somewhere and I just need someone to help me.  Can you help me decode
or or lead me to someone who can help me, please :(

**Pull request merged**